### ISSUE TITLE
Gerald Stopped Cutting Mid Job and LCD Screen went blank.

### ISSUE THREAD

**Issue by Draughtworx** [DoESLiverpool/somebody-should#1542]
Whilst in the middle of a small cutting job last night Gerald just stopped and the LCD screen turned blank. I restarted a couple of times. There is no response to the screen, the rise and lower bed buttons make a small action but don't work to rise or lower the bed. I'm without clue as to why. 

Does anybody have any suggestions?

**Comment by johnmckerrell** [comment]
@DoESLiverpool/laser-maintenance any ideas?

**Comment by JackiePease** [comment]
![IMG_20210114_092631](https://user-images.githubusercontent.com/12802256/104571142-aa51fa80-564a-11eb-8a4c-840b8cde8f14.jpg)
When I opened the side panel, the red light on the small power supply? In front was intermittently switching on and off in a random way. By the time I got my camera it was on constant.
The panel is still not working. If anyone has got anything I can try please get in touch.

**Comment by JackiePease** [comment]
![Uploading IMG_20210114_092619.jpg…]()

**Comment by magman2112** [comment]
This is possibly a failed power supply or something else faulty that is pulling the power supply down. Unfortunately, there are 2 power supplies if I remember correctly, one for the controller and one for the stepper drivers, so we don’t know which one is playing up, though I suspect the controller PSU (which may also explain some of the other issues we have had with Gerald lately)

If it is a faulty power supply, we will likely need to order a replacement, but we might be able to make a workaround for the short term.

I will try to get across to DoES either later this afternoon or this evening to do some diagnostics.

**Comment by JackiePease** [comment]
It seems to be the smaller one at the front

________________________________

**Comment by magman2112** [comment]
After a fair amount of diagnosis, @Sean-anotherone and I found that there was a short in one of the limit switches that pulled the power rail down via the controller.

as the limit switches aren’t working anyway at the moment, we simply left this unplugged till we do the major overhaul on Gerald.

As part of our diagnostics, we tried the spare controller that we have in the laser spares cupboard. This was found to work well and in fact the main board is now installed in Gerald along with the original controller card.

For future reference, the smaller power supply provides 24V to the controller card, which is then converted to 5V via a DC/DC converter module on the controller card. The larger power supply powers the stepper drivers.

We then checked the alignment, which was good, but Sean also found that the air assist tubing at one point was intruding in the laser path. A quick addition of a tie wrap and this issue was resolved and Gerald was back to cutting 3mm ply at 18mm/70% power.
